discovery-provider/src/api/v1/playlists.py

```diff
@@ -328,48 +335,42 @@ class FullTrendingPlaylists(Resource):
         }
     )
 
-    def get_cache_key(self):
-        request_items = to_dict(request.args)
-        request_items.pop('limit', None)
-        request_items.pop('offset', None)
-        key = extract_key(request.path, request_items.items())
-        return key
-
     @record_metrics
     @full_ns.marshal_with(full_trending_playlists_response)
     def get(self):
         """Get trending playlists"""
-        # Parse args
         args = full_trending_parser.parse_args()
-        offset, limit = format_offset(args), format_limit(args, TRENDING_LIMIT)
-        current_user_id, time = args.get("user_id"), args.get("time", "week")
-        time = "week" if time not in ["week", "month", "year"] else time
+        strategy = trending_selector.get_strategy(TrendingType.PLAYLISTS)
+        playlists = trending_playlists(request, args, strategy)
```

Comment:
Nit: I like that you pulled this out! Could we call it `get_trending_playlists`? I think the verb helps clarify

Reply:
Actually I see why you can't name it that (conflicts). Maybe if we combine the endpoints we don't need to pull this logic out at all?